CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+Contributing to WildFly Elytron
+==================================
+- [Forking the Project](#forking-the-project)
+- [Issues](#issues)
+    * [Good First Issues](#good-first-issues)
+- [Setting up your Developer Environment](#setting-up-your-developer-environment)
+- [Coding Guidlines](#coding-guidlines)
+
+
+## Forking the Project 
+To contribute, you will first need to fork the [wildfly-elytron](https://github.com/wildfly-security/wildfly-elytron) repository. 
+
+This can be done by looking in the top-right corner of the repository page and click "Fork".
```

Comment:
Not sure, but shouldn't `click` be changed to `clicking`? If this is grammatically correct then nevermind my comment

Reply:
+1, Diana, this should be clicking

Reply:
Changed